ipc/large-vector-allocate-failure-crash.html should be a macOS only test

gavin-apple opened the pull request:
#### 8eb8798cbd6ab8d51fdf974fb1a6b45fd59a1678
<pre>
ipc/large-vector-allocate-failure-crash.html should be a macOS only test
<a href="https://bugs.webkit.org/show_bug.cgi?id=289620">https://bugs.webkit.org/show_bug.cgi?id=289620</a>
<a href="https://rdar.apple.com/146870032">rdar://146870032</a>

Reviewed by Alex Christensen.

ipc/large-vector-allocate-failure-crash.html tests the WebPaymentCoordinatorProxy subsystem.
This previously ran in the UI process on all platforms but now only runs there on macOS.
Update the test expectations to account for this.

* LayoutTests/TestExpectations:
* LayoutTests/ipc/mac/large-vector-allocate-failure-crash.html: Renamed from LayoutTests/ipc/large-vector-allocate-failure-crash.html.
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/mac-site-isolation/TestExpectations:
* LayoutTests/platform/mac-wk2/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/292496@main">https://commits.webkit.org/292496@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/e15668f10cc67ab2ebdae028a93fd2ef032a46d5

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/96029 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/15643 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/5591 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/101091 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/46538 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/15938 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/24075 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/73206 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/30432 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/99032 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/11933 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/86735 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/53543 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/11677 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/4487 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/45873 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/81824 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/4588 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/103117 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/23096 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/16824 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/82248 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/23347 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/82749 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/81620 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/20525 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/26222 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/3655 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/16451 "Built successfully") | 
| [❌ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/15475 "") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/23059 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/22718 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/26198 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/24459 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/31a9de7b4b888510c1d08d74125a05300a096a5d. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/42329

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/50f22125291dff534d7be3ad3751640a30002bff. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/42329

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/6c492045c8be96c106b93db98e4153a7c7927567. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/42329

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/25352e677bae4862d682f7bc7bb45f88d627893e. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/42329

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/e15668f10cc67ab2ebdae028a93fd2ef032a46d5. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/42329

webkit-commit-queue commented:
Commit message contains (OOPS!) and Marta Darbinyan is not a reviewer, blocking PR #42329. Details: [Build #20511](https://ews-build.webkit.org/#/builders/19/builds/20511)

webkit-commit-queue commented:
Commit message contains (OOPS!) and Marta Darbinyan is not a reviewer, blocking PR #42329. Details: [Build #15475](https://ews-build.webkit.org/#/builders/22/builds/15475)

webkit-commit-queue commented:
Commit message contains (OOPS!) and Marta Darbinyan is not a reviewer, blocking PR #42329. Details: [Build #20512](https://ews-build.webkit.org/#/builders/19/builds/20512)

gavin-apple committed:
ipc/large-vector-allocate-failure-crash.html should be a macOS only test
https://bugs.webkit.org/show_bug.cgi?id=289620
rdar://146870032

Reviewed by Alex Christensen.

ipc/large-vector-allocate-failure-crash.html tests the WebPaymentCoordinatorProxy subsystem.
This previously ran in the UI process on all platforms but now only runs there on macOS.
Update the test expectations to account for this.

* LayoutTests/TestExpectations:
* LayoutTests/ipc/mac/large-vector-allocate-failure-crash.html: Renamed from LayoutTests/ipc/large-vector-allocate-failure-crash.html.
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/mac-site-isolation/TestExpectations:
* LayoutTests/platform/mac-wk2/TestExpectations:

Canonical link: https://commits.webkit.org/292496@main

webkit-commit-queue commented:
Committed 292496@main (8eb8798cbd6a): <https://commits.webkit.org/292496@main>

Reviewed commits have been landed. Closing PR #42329 and removing active labels.
